packages/kbn-search-api-panels/components/preprocess_data.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiThemeProvider,
+  EuiTitle,
+  EuiToken,
+  EuiPanel,
+  EuiLink,
+  EuiFlexGrid,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const PreprocessDataPanel: React.FC = () => {
+  return (
+    <EuiThemeProvider colorMode="dark">
+      <EuiPanel paddingSize="xl">
+        <EuiFlexGrid direction="column" columns={2} gutterSize="xl">
+          <EuiFlexItem>
+            <EuiFlexGroup alignItems="flexStart" justifyContent="flexStart">
+              <EuiFlexItem grow={false}>
+                <EuiToken iconType="sparkles" size="l" color="#466B8D" />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiTitle size="s">
+                  <h3>
+                    {i18n.translate(
+                      'searchApiPanels.preprocessData.overview.dataEnrichment.title',
+                      {
+                        defaultMessage: 'Data enrichment',
+                      }
+                    )}
+                  </h3>
+                </EuiTitle>
+                <EuiSpacer size="s" />
+                <EuiText size="s">
+                  <p>
+                    {i18n.translate(
+                      'searchApiPanels.preprocessData.overview.dataEnrichment.description',
+                      {
+                        defaultMessage:
+                          'Add information from external sources or apply transformations to your documents for more contextual, insightful search.',
+                      }
+                    )}
+                  </p>
+                </EuiText>
+                <EuiSpacer size="s" />
+                <EuiText>
+                  <p>
+                    <EuiLink href={'#'} target="_blank">
```

Comment:
Given this is for serverless we will need to populate these thinks before we can merge. We have to assume any commit can go to production.

Reply:
Yes. I agree

Reply:
updated all links  - d9281e61e809e6c8104d6efafb18539afa49e2e7